packages/fcl-wc/src/fcl-wc.ts

```diff
@@ -13,6 +13,7 @@ export interface FclWalletConnectConfig {
   wcRequestHook?: any
   pairingModalOverride?: any
   wallets?: any[]
+  showNotifications?: boolean
```

Comment:
Do you want them on by default so this is switch to disabled or this is how you want it? Just curious.